.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["es2015"]
+}
```

Comment:
@marcobiedermann This [can be added in the `package.json` file](https://babeljs.io/docs/usage/babelrc/#use-via-package-json) as:

```json
"babel": {
  "presets": [
    "es2015"
  ]
}
```